src/cpp/cpp_convert_type.cpp

```diff
@@ -130,6 +130,10 @@ void cpp_convert_typet::read_rec(const typet &type)
     constexpr_cnt++;
   else if(type.id()==ID_extern)
     extern_cnt++;
+  else if(type.id()==ID_noreturn)
+  {
+    noreturn_cnt++;
```

Comment:
Nit: prefer prefix increment for consistency 
```c++
++noreturn_cnt;
```

Reply:
But ... that wouldn't be _consistent_ here?